packages/remark-parse/lib/tokenizer.js

```diff
@@ -6,27 +6,23 @@
  * @fileoverview Markdown tokenizer.
  */
 
-'use strict';
+export default factory;
```

Comment:
Yeah I Googled too, but couldn’t find it out.

> It was my understanding that the export keyword allocation is meant to be immutable...

That’s right, but I’m unsure whether function hoisting is “mutable” though 😬